test/basics.js

```diff
@@ -0,0 +1,92 @@
+require('./init.js');
+var should = require('should');
+var ds, COSModel;
+
+
+describe('cos model creation', function() {
+	before(function(done){
+		ds = global.getDataSource(); //.settings is your config
+
+		COSModel = ds.define('COSModel', {
+			name: {type: String},
+			age: {type: Number},
+		});
+		ds.automigrate(done);
+	});
+
+	describe('model data gets updated properly', function() {
+		var cos1, cos2;
+		before('create entry', function(done) {
+			this.timeout(0);
+
+
+			COSModel.create({
+				name: 'Conor',
+				age: 23,
+				shopping: ['apple','orange'],
+			}, function(err, cosmod) {
```

Comment:
I think it's better to use Promise API in the tests to make the code shorter (no need to check for error after each async operation).

```js
return COSModel.create({
  // ...
})
  .then(cosmod => {
    cos1 = cosmod;
    return COSModel.create({
      // ...
    });
  })
  .then(cosmod2 => {
    // etc.
  });
```